conf/spark-env.sh

```diff
@@ -87,6 +87,14 @@ else
     echo "spark-env: StatsD metrics require Mesos UCR. For dispatcher metrics, enable the 'UCR_containerizer' option. For driver metrics, include '--conf spark.mesos.containerizer=mesos' in your run"
 fi
 
+if [ -n "${STATSD_UDP_HOST}" ] && [ -n "${STATSD_UDP_PORT}" ]; then
+    sed -e "s/<STATSD_UDP_HOST>/${STATSD_UDP_HOST}/g" \
```

Comment:
I know this is a backport PR, but would it make sense to at some point use mustache templating instead of non-standard `<VAR>` tags?

Reply:
For that matter, don't we have `bootstrap` in this image to do the templating?

Reply:
We use `bootstrap` to apply templating to config files in other data services. 

Look at https://github.com/mesosphere/dcos-commons/blob/master/sdk/bootstrap/main.go#L338 for example. Note that we may want to add a filename parameter to the utility to allow for local usage instead of mapping this through the envvars.